actionmailer/test/test_helper_test.rb

```diff
@@ -383,7 +389,7 @@ def test_assert_enqueued_email_with_with_no_block_with_args
     end
   end
 
-  def test_assert_enqueued_email_with_with_parameterized_args
+  def test_assert_enqueued_email_with_parames
```

Comment:
Is this misspelled? Should it be `...with_params`?

```suggestion
  def test_assert_enqueued_email_with_params
```